lang/constants.ts

```diff
@@ -0,0 +1,104 @@
+import { bold } from 'chalk';
+export const doctor = {
+  command: {
+    describe:
+      'Retrieve diagnostic information about your local HubSpot configurations.',
+    options: {
+      outputDir: 'Directory to save a detailed diagnosis JSON file in',
+    },
+    errors: {
+      generatingDiagnosis: 'Error generating diagnosis',
+      unableToWriteOutputFile: (file: string, errorMessage: unknown) =>
+        `Unable to write output to ${bold(file)}, ${errorMessage}`,
```

Comment:
Instead of using the bespoke markdown language we could directly use JS functions and directly invoke whatever chalk methods we wish. We also get the benefits of type checking for these string generating methods

Reply:
To me, this is one of the most convincing arguments for moving toward a constant file with JS functions to implement interpolations. The functions are more readable to me than our bespoke markdown language (love that phrasing).